rebar.config

```diff
@@ -89,7 +89,12 @@
 
         % waiting for https://github.com/szktty/erlang-lz4/pull/12 to be merged
         {lz4, {git, "https://github.com/aeternity/erlang-lz4.git",
-              {ref, "1ff9f36"}}}
+              {ref, "1ff9f36"}}},
+
+        % for lightweight trace-based debugging
+        {trace_runner, {git, "git://github.com/uwiger/trace_runner.git",
```

Comment:
Why not only in specific test profiles?

Reply:
@lucafavatella Because the code needs to be instrumented (ideally using the `event/1` function rather than debug logging, which is supported by the `trace_runner.hrl`.)

While it is possible to instrument the code without having `trace_runner` included, having it included also makes it possible for dialyzer to check the instrumentation callbacks.